examples/tensorflow-comparison/comparison-training-singlenode.ipynb

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
This should have titles, words, and comments. It shouldn't just be a notebook that's empty